ci-docker-base/Dockerfile.cilint-clang-tidy

```
@@ -1,12 +1,21 @@
 # ubuntu18.04-cuda10.2-py3.6-tidy11
 FROM nvidia/cuda:10.2-devel-ubuntu18.04
 
-RUN apt-get update && apt-get upgrade -y
-RUN apt install -y software-properties-common wget
-RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
-RUN apt-add-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-11 main"
-RUN apt-add-repository ppa:git-core/ppa
-
-RUN apt-get update && apt-get upgrade -y && apt-get install -y git python3-dev python3-pip build-essential cmake clang-tidy-11 time
-RUN update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/bin/clang-tidy-11 1000
+# Copy diffs
+COPY ./clang-tidy-checks clang-tidy-checks
+
+# Install dependencies
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y --no-install-recommends \
+    git python3-dev python3-pip build-essential time \
+    cmake clang lld ninja-build
+
+# Run setup script (See ./clang-tidy-checks/README.md for more details)
+RUN cd ./clang-tidy-checks && ./setup.sh
+
+# Link clang-tidy to custom build
+RUN ln -s $(pwd)/clang-tidy-checks/llvm-project/build/bin/clang-tidy /bin/clang-tidy
```

can you add a smoke-test type thing just to ensure that the build installed everything successfully? afaik `grep` should return `1` if it doesn't find any lines

```suggestion
RUN ln -s $(pwd)/clang-tidy-checks/llvm-project/build/bin/clang-tidy /bin/clang-tidy

RUN clang-tidy --list-checks | grep misc-max-tokens
```